js_modules/dagster-ui/packages/ui-core/src/assets/useRecentAssetEvents.tsx

```diff
@@ -79,6 +79,8 @@ export function useRecentAssetEvents(
       xAxis,
     };
   }, [data, loading, refetch, loadUsingPartitionKeys, xAxis]);
+
+  return value;
```

Comment:
This was weird -- TypeScript was regarding the returned value as `any`, but determines a correct type for the const.

Reply:
That does sound weird????